### PR TITLE
Issue #2376: Change 'View mode' to 'Display mode' to avoid confusion …

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2376

### CODE REVIEW

File: core/modules/entity/entity.api.php [hunk 1]
```diff
@@ -80,27 +80,28 @@
...
- *     track of. Keys of the array are view mode names. Each view mode is
- *     described by an array with the following key/value pairs:
- *     - label: The human-readable name of the view mode
- *     - custom settings: A boolean specifying whether the view mode should by
- *       default use its own custom field display settings. If FALSE, entities
- *       displayed in this view mode will reuse the 'default' display settings
- *       by default (e.g. right after the module exposing the view mode is
- *       enabled), but administrators can later use the Field UI to apply custom
- *       display settings specific to the view mode.
+ *   - display modes: An array describing the display modes for the entity type.
```
Comment (quicksketch) [inline core/modules/entity/entity.api.php]: The key in the array is still "view modes" in this function, so this documentation is incorrect.

File: core/modules/field/field.module [hunk 2]
```diff
@@ -866,20 +866,20 @@ function field_get_display($instance, $view_mode, $entity) {
 }
 
 /**
- * Returns the display settings to use for pseudo-fields in a given view mode.
+ * Returns the display settings to us for pseudo-fields in a display mode.
```
Comment (quicksketch) [inline core/modules/field/field.module]: `us` should stay as `use`

**Review by quicksketch — CHANGES_REQUESTED** [review]

File: core/modules/entity/entity.api.php (hunk 1 above)
Comment (jenlampton) [reply core/modules/entity/entity.api.php]: fixed.

File: core/modules/field/field.module (hunk 2 above)
Comment (jenlampton) [reply core/modules/field/field.module]: fixed.

**Comment by jenlampton** [comment]
Pushed a new PR with those docs changes.

**Review by quicksketch — APPROVED** [review]

**Comment by quicksketch** [comment]
All tests passing and marked RTBC in the queue. Looks great! Merged into 1.x for 1.8.0.

**Pull request merged**